_infra/helm/securebanking-openbanking-uk-rcs/values.yaml

```diff
@@ -1,4 +1,7 @@
 deployment:
+  # Affinity Configuration
+  # Uncomment and add in required configuration 
+  # affinity:
```

Comment:
Have you tried:
```
affinity: {}
```

Example in nginx chart:
https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml#L218C3-L218C15
https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/default-backend-deployment.yaml#L116-L118

I'd rather not have commented out stuff where possible